packages/upset/src/components/Columns/Attribute/AttributePlots/DotPlot.tsx

```diff
@@ -25,6 +26,29 @@ export const DotPlot: FC<Props> = ({
     return null;
   }
 
+  /**
+   * Generates a y offset for the provided index.
+   * Seeded based on row size so that jitter is consistent between renders, and also varies between rows.
+   * Rows of the same size will have the same jitter.
```

Comment:
Have you considered seeding it based on the row ID rather than size so that rows are always different?

Reply:
Using the row id as well, these two rows have the same size, but different jitter:

![image](https://github.com/visdesignlab/upset2/assets/35744963/53bd8164-a505-462e-b27f-57753746f7ad)

Reply:
That looks pretty good. ATM I also have a util function in my backselection PR that can hash a string, but probably not necessary here.